packages/next/src/export/index.ts

```diff
@@ -447,35 +450,31 @@ export default async function exportApp(
     }
 
     // Start the rendering process
-    const renderOpts = {
-      dir,
+    const renderOpts: WorkerRenderOptsPartial = {
+      previewProps: prerenderManifest?.preview,
       buildId,
       nextExport: true,
       assetPrefix: nextConfig.assetPrefix.replace(/\/$/, ''),
       distDir,
       dev: false,
-      hotReloader: null,
       basePath: nextConfig.basePath,
       canonicalBase: nextConfig.amp?.canonicalBase || '',
-      ampValidatorPath: nextConfig.experimental.amp?.validator || undefined,
       ampSkipValidation: nextConfig.experimental.amp?.skipValidation || false,
       ampOptimizerConfig: nextConfig.experimental.amp?.optimizer || undefined,
       locales: i18n?.locales,
       locale: i18n?.defaultLocale,
       defaultLocale: i18n?.defaultLocale,
       domainLocales: i18n?.domains,
-      trailingSlash: nextConfig.trailingSlash,
```

Comment:
Is `trailingSlash` really not needed? <picture data-single-emoji=":thonkfast:" title=":thonkfast:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thonkfast/474078c6ec6b8cbc.gif" alt=":thonkfast:" align="absmiddle"></picture>

Reply:
It's not actually part of the types (`RenderOpts` strikes again), but it is used by the export code, that's why it's still on the input type:

https://github.com/vercel/next.js/blob/2da0b8a17817415395cc1e418ca86fd16513a3fb/packages/next/src/export/worker.ts#L54

It's added here:

https://github.com/vercel/next.js/blob/2da0b8a17817415395cc1e418ca86fd16513a3fb/packages/next/src/export/index.ts#L720